core/src/main/resources/META-INF/resources/primefaces-extensions/timeago/0-jquery.timeago.js

```diff
@@ -42,7 +42,7 @@
     settings: {
       refreshMillis: 60000,
       allowPast: true,
-      allowFuture: false,
+      allowFuture: true, // PFE #1112
```

Comment:
@melloware this was the easiest fix. IMO it makes sense to set this as the global default. If you don't this would display "The next month is 29 days ago", which is a bug in my book.